onnxruntime/core/framework/tensor_type_and_shape.cc

```diff
@@ -168,6 +131,15 @@ ONNXTensorElementDataType TensorDataTypeToOnnxRuntimeTensorElementDataType(
   return type;
 }
 
+ONNXTensorElementDataType MLDataTypeToOnnxRuntimeTensorElementDataType(
+    const onnxruntime::DataTypeImpl* cpp_type) {
+  auto prim_type = cpp_type->AsPrimitiveDataType();
```

Comment:
Can cpp_type be a reference?

Reply:
I don't think so.

---
In reply to: [341919030](https://github.com/microsoft/onnxruntime/pull/2307#discussion_r341919030) [](ancestors = 341919030)